e2e/qgb/qgb_network.go

```diff
@@ -250,12 +293,15 @@ func (network QGBNetwork) StartBase() error {
 	return nil
 }
 
-func (network QGBNetwork) WaitForNodeToStart(rpcAddr string) error {
-	timeoutChan := time.After(5 * time.Minute)
+func (network QGBNetwork) WaitForNodeToStart(_ctx context.Context, rpcAddr string) error {
+	ctx, _ := context.WithTimeout(_ctx, 5*time.Minute) //nolint:govet
```

Comment:
no need for a cancel function, we only need the timeout. If the happy path executes, then it will just return.